src/site/content/en/handbook/style/index.md

```diff
@@ -0,0 +1,158 @@
+
+## Keep it simple
+Focus on essential details when explaining a tool or technique. Avoid overwhelming the reader with "real-world" examples that include complexities that aren't required to understand the basic concept. Save those details for supplemental codelabs or follow-up posts. See the "[Code splitting with React.lazy and Suspense](/code-splitting-suspense)" post for an example.
+
+Avoid jargon. If you absolutely _must_ use a term most web.dev readers are unlikely to know, define it in a short aside. See the "[web.dev components](/handbook/web-dev-components#asides)" post for how to include asides. Also check out the "[Inclusion and accessibility](/handbook/inclusion-and-accessibility#use-readable-language)" post for more info about writing readable language.
+
+{% Compare 'worse', 'Don’t' %}
+> In some scripts, graphemes can be visually joined when they're adjacent.
+
+_Script_ and _grapheme_ are very precise, but they're likely to be unfamiliar to most readers and may cause confusion since readers probably more commonly see _script_ referring to a piece of code.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> In some written languages, certain characters can be visually joined when they're adjacent.
+
+While _written language_ and _character_ are slightly less precise, they should be more familiar and less likely to cause misunderstandings.
+{% endCompare %}
+
+## Be accurate
+Make sure your information aligns to current documentation, standards, and best practices. If you're writing about a topic that isn't in your area of expertise (or even if it is!), get input from other experts.
+
+## Give useful context
+Tell readers what you're expecting them to know up front—and provide links to where they can learn it if you can. See the "[Get started: optimize your React app](/get-started-optimize-react/#what-will-you-learn)" post for an example.
+
+Explain why what you're teaching matters before jumping into the instruction. This context can be provided in a collection via an introductory post (see the "[Accessible to all](/accessible)" collection for an example) or in a post via an introductory paragraph (the "[Use Imagemin to compress images](/use-imagemin-to-compress-images)" post is a nice example).
+
+Indicate which details are relevant in longer code blocks by providing explanations and code highlighting. If readers don't need to understand everything about the code, tell them that—it makes the task less intimidating, especially for less experienced developers. See the "[Serve responsive images](/serve-responsive-images)" post for an example.
+
+When there's more than one way to solve a problem, organize them in order of simplest to most complex.
+
+## Be mindful of the development landscape
+Acknowledge browser complexity. Clearly state browser support (e.g., link to the relevant feature page on [caniuse](https://www.caniuse.com/)) and provide workarounds for browser incompatibilities (e.g., polyfills).
```

Comment:
Don't use caniuse. [MDN tables](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorker#Browser_compatibility) is more detailed. Both are crowdsourced, but MDN has a formal plan for data completeness and engineering resources devoted to that goal. There is, I believe, polyfill site, which I can't find at the moment.